types/fast-als/index.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for fast-als 1.0
+// Project: https://github.com/thorough-developer/fast-als
+// Definitions by: Elias Fauser <https://github.com/elias-fauser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
```

Comment:
```suggestion
```
why this one is needed? (we're trying to avoid overload dependencies between DT packages)

Reply:
Sry this was a copy and paste error. Removed it